src/hooks/charts/useChartData.ts

```diff
@@ -82,6 +85,7 @@ export default function useChartData(asset: any, secondStore: any) {
     [setParams]
   );
 
+  // FIXME: check if native currency is correct
```

Comment:
can remove this or can add the following so I can take a look

`TODO: @skylarbarrera APP-211`